pkgs/by-name/zu/zulip/package.nix

```
@@ -1,41 +1,78 @@
 { lib
-, fetchurl
-, appimageTools
+, fetchFromGitHub
+, buildNpmPackage
+, electron_32
+, makeDesktopItem
+, makeShellWrapper
+, copyDesktopItems
 }:
 
-let
+buildNpmPackage rec {
   pname = "zulip";
   version = "5.11.1";
 
-  src = fetchurl {
-    url = "https://github.com/zulip/zulip-desktop/releases/download/v${version}/Zulip-${version}-x86_64.AppImage";
-    hash = "sha256-t5qBm5+kTdeRMvcHpNbS5mp184UG/IqgJrtj7Ntcbb0=";
-    name="${pname}-${version}.AppImage";
+  src = fetchFromGitHub {
+    owner = "zulip";
+    repo = "zulip-desktop";
+    rev = "v${version}";
+    hash = "sha256-ELuQ/K5QhtS4QiTR35J9VtYNe1qBrS56Ay6mtcGL+FI=";
   };
 
-  appimageContents = appimageTools.extractType2 {
-    inherit pname version src;
-  };
+  npmDepsHash = "sha256-13Rlqa7TC2JUq6q1b2U5X3EXpOJGZ62IeF163/mTo68=";
+
+  env.ELECTRON_SKIP_BINARY_DOWNLOAD = 1;
```

```suggestion
  env = {
    ELECTRON_SKIP_BINARY_DOWNLOAD = 1;
  };
```

No reason to introduce skew vs. the cherry-picked commit here.